src/tests/multihost/alltests/test_default_debug_level.py

```diff
@@ -204,3 +213,182 @@ def test_0005_bz1915319(self, multihost, backupsssdconf):
+                                    'groupOfNames'.encode('utf-8')],
+                    'member': f'uid=tempuser3,{ds_suffix}'.encode('utf-8')}
+        ldap_inst.add_entry(grp_info, group_dn)
+        group_dn = f"cn=Group22,ou=Groups,{ds_suffix}"
+        grp_info = {'cn': 'Group22'.encode('utf-8'),
+                    'gidNumber': '222022'.encode('utf-8'),
+                    'objectClass': ['top'.encode('utf-8'),
+                                    'posixGroup'.encode('utf-8'),
+                                    'groupOfNames'.encode('utf-8')],
+                    'member': f'uid=tempuser3,{ds_suffix}'.encode('utf-8')}
+        ldap_inst.add_entry(grp_info, group_dn)
+        group_dn = f"cn=Group222,ou=qagroup,{ds_suffix}"
+        grp_info = {'cn': 'Group222'.encode('utf-8'),
+                    'gidNumber': '222000'.encode('utf-8'),
+                    'objectClass': ['top'.encode('utf-8'),
+                                    'posixGroup'.encode('utf-8'),
+                                    'groupOfNames'.encode('utf-8')],
+                    'member': [f'cn=Group11,ou=qagroup,{ds_suffix}'.encode('utf-8')]}
+        ldap_inst.add_entry(grp_info, group_dn)
+        group_dn = f"cn=Group111,ou=Groups,{ds_suffix}"
+        grp_info = {'cn': 'Group111'.encode('utf-8'),
+                    'gidNumber': '111000'.encode('utf-8'),
+                    'objectClass': ['top'.encode('utf-8'),
+                                    'posixGroup'.encode('utf-8'),
+                                    'groupOfNames'.encode('utf-8')],
+                    'member': [f'cn=Group222,ou=qagroup,{ds_suffix}'.encode('utf-8'),
+                               f'cn=Group22,ou=Groups,{ds_suffix}'.encode('utf-8'),
+                               f'uid=tempuser2,{ds_suffix}'.encode('utf-8')]}
+        ldap_inst.add_entry(grp_info, group_dn)
+        tools.sssd_conf("domain/example1",
+                        {'cache_credentials': True,
+                         'enumerate': True,
+                         'ldap_search_base': f'ou=Groups,{ds_suffix}',
+                         'ldap_schema': 'rfc2307bis',
+                         'ldap_group_object_class': "groupOfNames"}, action='update')
+        tools.clear_sssd_cache()
+        client.run_command("getent -s sss group")
+        client.run_command("getent -s sss group > /tmp/grp_file")
+        for dn_dn in [f'uid=tempuser2,{ds_suffix}',
+                      f'uid=tempuser3,{ds_suffix}',
+                      f'cn=Group111,ou=Groups,{ds_suffix}',
+                      f'cn=Group222,ou=qagroup,{ds_suffix}',
+                      f'cn=Group22,ou=Groups,{ds_suffix}',
+                      f'cn=Group11,ou=qagroup,{ds_suffix}',
+                      f'ou=qagroup,{ds_suffix}']:
+            ldap_inst.del_dn(dn_dn)
+        find_logs(multihost, "/tmp/grp_file", "Group111")
+        find_logs(multihost, "/tmp/grp_file", "Group22")
+
+    @staticmethod
+    def test_bz785898(multihost, backupsssdconf):
```

Comment:
Could you share the original bash test location? I'm not able to locate a testcase with mentioned bz or title in bash code. The link[1] in description is not this testcase.

1 https://gitlab.cee.redhat.com/sssd/sssd-qe/-/blob/RHEL8.8/client/ldap_provider/ldap_id_ldap_auth/bugzilla-automation.sh#L300